server/enumeration/licenceStatus.ts

```diff
@@ -16,4 +16,15 @@ enum LicenceStatus {
   OOS_RECALL = 'OOS_RECALL',
 }
 
+const SelectableLicenceStatus = Object.values(LicenceStatus).filter(
```

Comment:
think this should be lower camel case as its data not a Type

Reply:
now replaced with snake-case